flujo/steps/cache_step.py

```
@@ -122,12 +122,37 @@ def _serialize_for_cache_key(
+                            if isinstance(fields, dict) and fields:
+                                field_names = [str(k) for k in fields.keys()]
+                        except Exception:
+                            pass
+
+                    out: dict[str, Any] = {}
+                    for k in field_names:
+                        try:
+                            out[k] = _serialize_for_cache_key(
+                                getattr(obj, k), visited, _is_root=False
+                            )
+                        except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)